docs/administration/database.md

```diff
@@ -1,4 +1,362 @@
+
+For that purpose, you should specify some environment variables to the container startup command to make it work. The table below lists these needed variables:
+  Variable         | Mandatory    |  Default value   |    Description
+  -----------------|--------------|------------|---------------------------
+  EXO_DB_TYPE      |  No          |hsqldb      | The database type to be used, supports hsqldb (not for production) and mysql databases.
+  EXO_DB_HOST      |  No          | mysql      | The host to connect to the database server.
+  EXO_DB_PORT      |  No          | 3306       | The port to connect to the database server.
+  EXO_DB_NAME      |  No          |  exo       | The name of the database schema to use.
+  EXO_DB_USER      |  No          | exo        |   The username to connect to the database.
+  EXO_DB_PASSWORD  |  Yes         |            |   The password to connect to the database.
+  
+An example of the execution command to use MySQL database for eXo Platform docker image:
+
+```shell  
+    docker run -d \
+        -p 8080:8080 \
+        -e EXO_DB_TYPE="mysql" \
+        -e EXO_DB_HOST="mysql.server-hostname.org" \
+        -e EXO_DB_USER="exo" \
+        -e EXO_DB_PASSWORD="my-secret-pw" \
+        exoplatform/exo-community
+```
+
+For more details, you can look at this [documentation](https://hub.docker.com/r/exoplatform/exo-community/).
+
+## Datasource JNDI name
+
+As said previously, eXo Platform uses two datasources, *exo-idm_portal* and *exo-jcr_portal*. If for any reason you change those names in datasource configuration (xml file), you need to match them in some other files.
+
+::: tip
+
+The properties file (`exo.properties`) will not take **_portal** suffix as it is appended automatically by eXo, as detailed below.
+:::
+
+There is a constraint that the suffix of datasource JNDI names must be **_portal**. Take JCR as example, it uses the following property:
+
+```properties
+    exo.jcr.datasource.name=java:/comp/env/exo-jcr
+```
+
+to look up a datasource for the portal. Because the core of eXo Platform is designed for supporting multi-portal, there are theoretically different datasources for different portals. Consequently this property is treated as datasource name\'s prefix, and the portal name (knowing that it is \"portal\" by default) is appended to complete the name in JNDI lookup.
+
+So if you change the JDNI names *exo-idm_portal* and **exo-jcr_portal**, you need to edit the following properties in `gatein/conf/exo.properties`:
+
+```properties  
+  # JNDI Name of the IDM datasource
+  exo.idm.datasource.name=java:/comp/env/exo-idm
+  ...
+  # name of the datasource that will be used by eXo JCR
+  exo.jcr.datasource.name=java:/comp/env/exo-jcr
+```
+
+You also need to edit `conf/Catalina/localhost/context.xml.default` file:
+
```

Comment:
I'm sure this part is pertinent.
What do you think ?

Reply:
@rdenarie this is for informative purpose and won't harm to put it there. I migrated the original content, and I think it s worth knowing as a side note

Reply:
I dont want to make the documentation too complex to read and understand, and this part add complexity.
For us, we are knowing this and we understand it. For a newbie, it could be complex to understand, that's why I prefer to remove it

Reply:
Ok